tests/src/main/java/org/hibernate/beanvalidation/tck/tests/constraints/constraintdefinition/ConstraintDefinitionsTest.java

```diff
@@ -75,6 +77,36 @@ public void testConstraintWithCustomAttributes() {
 	}
 
 	@Test
+	@SpecAssertions({
+			@SpecAssertion(section = "3.2", id = "b")
+	})
+	public void testRepeatableConstraint() {
+
```

Comment:
Yes, I mimicked the other methods - it was before we discussed it and choose to get rid of them. Fixed.